pkg/kubelet/network/hostport/hostport_syncer_test.go

```diff
@@ -168,16 +168,16 @@ func TestOpenPodHostports(t *testing.T) {
 		false,
 	}
 
-	err := h.OpenPodHostportsAndSync(tests[0].mapping, "br0", activePodPortMapping)
+	err := h.OpenPodHostportsAndSync(tests[0].mapping, activePodPortMapping)
 	if err != nil {
 		t.Fatalf("Failed to OpenPodHostportsAndSync: %v", err)
 	}
 
 	// Generic rules
 	genericRules := []*ruleMatch{
-		{-1, "POSTROUTING", "-m comment --comment \"SNAT for localhost access to hostports\" -o br0 -s 127.0.0.0/8 -j MASQUERADE"},
-		{-1, "PREROUTING", "-m comment --comment \"kube hostport portals\" -m addrtype --dst-type LOCAL -j KUBE-HOSTPORTS"},
-		{-1, "OUTPUT", "-m comment --comment \"kube hostport portals\" -m addrtype --dst-type LOCAL -j KUBE-HOSTPORTS"},
+		{-1, "PREROUTING", "-m comment --comment \"maybe kube hostport\" -m addrtype --dst-type LOCAL -j KUBE-LOCALDEST"},
+		{-1, "OUTPUT", "-m comment --comment \"maybe kube hostport\" -m addrtype --dst-type LOCAL -j KUBE-LOCALDEST"},
+		{-1, "KUBE-LOCALDEST", "-m comment --comment \"maybe kube hostport\" -d 127.0.0.1/32 -j KUBE-HOSTPORTS"},
```

Comment:
I thought the whole 127.0.0.0/8 was a local destination? Maybe I'm missing something...